### PR TITLE
Update tutorial list to mention Web API

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Review URL](https://review.docs.microsoft.com/en-us/aspnet/core/?branch=pr-en-us-3304#build-web-apis-and-web-ui-using-aspnet-core-mvc)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  can we put UID's on index pages so we don't have such weird links?
`See [Building web APIs](aspnet/core/tutorials/index.md#building-web-apis). `

**Comment by Rick-Anderson** [comment]
@tdykstra  I can't figure out the correct link for aspnetcore\tutorials\index.md

File: aspnetcore/index.md [hunk 1]
```diff
@@ -54,10 +54,11 @@ With ASP.NET Core you gain the following foundational improvements:
 
 ## Build web UI and web APIs using ASP.NET Core MVC
 
-* You can create well-factored and testable web apps that follow the Model-View-Controller (MVC) pattern. See [MVC](mvc/index.md) and [Testing](testing/index.md).
-
+* You can build HTTP services that reach a broad range of clients, including browsers and mobile devices.  ASP.NET Core is an ideal platform for building web APIs and RESTful apps on .NET Core. See [Building web APIs](aspnetcore/tutorials/index.md#building-web-apis).  
 * You can build HTTP services that support multiple formats and have full support for content negotiation. See [Formatting Response Data](mvc/models/formatting.md)
```
Comment (tdykstra) [inline aspnetcore/index.md]: this is now redundant, consider deleting this paragraph

File: aspnetcore/index.md [hunk 2]
```diff
@@ -54,10 +54,11 @@ With ASP.NET Core you gain the following foundational improvements:
...
 
+* You can create well-factored and testable web apps that follow the Model-View-Controller (MVC) pattern. See [MVC](mvc/index.md) and [Testing](testing/index.md).
```
Comment (tdykstra) [inline aspnetcore/index.md]: I'd put this paragraph first, or keep it here but change the section heading

File: aspnetcore/index.md [hunk 3]
```diff
@@ -54,10 +54,11 @@ With ASP.NET Core you gain the following foundational improvements:
 
 ## Build web UI and web APIs using ASP.NET Core MVC
 
-* You can create well-factored and testable web apps that follow the Model-View-Controller (MVC) pattern. See [MVC](mvc/index.md) and [Testing](testing/index.md).
-
+* You can build HTTP services that reach a broad range of clients, including browsers and mobile devices.  ASP.NET Core is an ideal platform for building web APIs and RESTful apps on .NET Core. See [Building web APIs](aspnetcore/tutorials/index.md#building-web-apis).
```
Comment (tdykstra) [inline aspnetcore/index.md]: ~~aspnetcore/~~ tutorials/index.md#building-web-apis 

And yes, there should be no problem with adding uid to index.md files.

File: aspnetcore/index.md (hunk 1 above)
Comment (Rick-Anderson) [inline aspnetcore/index.md]: Which part is redundant? the first paragraph is about web APIs, the 2nd formatting. I can try combining them.
Background: This a request from the PU to have a list of Web API tutorials - that's what I'm trying to work in.

Comment (Rick-Anderson) [inline aspnetcore/index.md]: Can you make the changes that I didn't fix and PR or merge into this?

**Comment by Rick-Anderson** [comment]
@balachir see the [Review URL](https://review.docs.microsoft.com/en-us/aspnet/core/?branch=pr-en-us-3304#build-web-ui-and-web-apis-using-aspnet-core-mvc)

File: aspnetcore/index.md (hunk 1 above)
Comment (tdykstra) [inline aspnetcore/index.md]: Will do

Comment (tdykstra) [inline aspnetcore/index.md]: @Rick-Anderson Updated the text to combine web api paragraphs, please review.

**Comment by balachir** [comment]
@Rick-Anderson the changes look good to me. Thanks!

**Pull request merged**